### ISSUE TITLE
Preserve sidebar scroll position when switching between views or reopening the sidebar

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#4783]
https://forums.zotero.org/discussion/119052/bug-reader-sidebar-tabs-reset-when-the-sidebar-is-closed

Note: It would be good to somehow preserve these and other reader states, even if the tab was unloaded during the current session. This is one of the reasons why I was thinking about moving reader loading/unloading into the reader itself. #4646 #3231

**Comment by adomasven** [comment]
XUL elements (and maybe others) have a mozilla framework handled `preserve` attribute where you can list attributes you want to be preserved and mozilla serializes them somewhere (in the profile?) as long as the elements have unique ID paths. Worth looking into, although you'd have to think about garbage collecting them for deleted items.

**Comment by mrtcode** [comment]
Good point. I believe we are already persisting the Reader sidebar width this way, but for scrolling, simply keeping the state in memory should be sufficient.
